bulldozer/update.go

```diff
@@ -59,59 +58,43 @@ func ShouldUpdatePR(ctx context.Context, pullCtx pull.Context, updateConfig Upda
 	return true, nil
 }
 
-func UpdatePR(ctx context.Context, pullCtx pull.Context, client *github.Client, updateConfig UpdateConfig, baseRef string) error {
+func UpdatePR(ctx context.Context, pullCtx pull.Context, client *github.Client, updateConfig UpdateConfig, baseRef string) {
 	logger := zerolog.Ctx(ctx)
 
-	//todo: should the updateConfig struct provide any other details here?
-
-	go func(ctx context.Context, baseRef string) {
-		ticker := time.NewTicker(2 * time.Second)
-		defer ticker.Stop()
-
-		for i := 0; i < MaxPullRequestPollCount; i++ {
```

Comment:
As far as I can tell, this loop and the ticker were dead code: no part of the loop body used `continue` and the final statement in the loop is `return`.